### PR TITLE
feature/searchbar

### PR DESCRIPTION
**Pull request by souzasmatheus**

### CODE REVIEW

File: src/components/common/molecules/SearchBar/index.tsx [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
-const SearchBar: FC<Props> = () => {
+const SearchBar = ({ onSubmit, placeholderText }: Props) => {
   return (
-    <div>
-      <h1>SearchBar</h1>
-      <FaSearch />
-    </div>
+    <Wrapper>
+      <Input
+        onKeyPress={(e: any) => {
```
Comment (souzasmatheus) [inline src/components/common/molecules/SearchBar/index.tsx]: @AllTwos I don't have a problem with small inline functions, but here you even have an if statement, so I would consider moving it upper. Something like `const handleKeyPress = (e) => ...`.
Also, the type of this event is `React.KeyboardEvent<HTMLInputElement>`. Please fix that.

File: src/components/common/molecules/SearchBar/styled.tsx [hunk 2]
```diff
@@ -1,43 +1,43 @@
-export {};
+import { FaSearch } from 'react-icons/fa';
+import styled from 'styled-components';
 
+const StyledMagnifierIcon = styled(FaSearch).attrs({})`
```
Comment (souzasmatheus) [inline src/components/common/molecules/SearchBar/styled.tsx:4]: This `.attrs({})` is doing nothing here, so we might as well remove it.

File: src/components/common/molecules/SearchBar/styled.tsx [hunk 3]
```diff
@@ -1,43 +1,43 @@
...
-}
+  border: 2px solid ${({ theme }) => theme.colors.grey.deepmist};
+  border-top-left-radius: 20px;
+  border-bottom-left-radius: 20px;
+  border-top-right-radius: 20px;
+  border-bottom-right-radius: 20px;
+`;
 
-.box input {
+const Input = styled.input<{ foo?: string }>`
```
Comment (souzasmatheus) [inline src/components/common/molecules/SearchBar/styled.tsx:27]: Please, remove this type annotation too (the whole `<{ foo?: string }>` thing). It was only there as an example lol

File: src/config/theme/index.ts [hunk 4]
```diff
@@ -23,6 +26,9 @@ const theme: DefaultTheme = {
   colors: {
     grey: {
       mercury: '#e6e6e6',
+      mist: '#fafafa',
+      deepmist: '#f0f0f0',
+      rockport: '#b7b7b7',
```
Comment (souzasmatheus) [inline src/config/theme/index.ts:31]: Great job adding the colors to the theme and typing them!

File: src/components/common/molecules/SearchBar/index.tsx (hunk 1 above)
Comment (AllTwos) [reply src/components/common/molecules/SearchBar/index.tsx]: Done

File: src/components/common/molecules/SearchBar/styled.tsx (hunk 2 above)
Comment (AllTwos) [reply src/components/common/molecules/SearchBar/styled.tsx:4]: I was just keeping this here for reference. Can delete if needed.

File: src/components/common/molecules/SearchBar/styled.tsx (hunk 3 above)
Comment (AllTwos) [reply src/components/common/molecules/SearchBar/styled.tsx:27]: I was just keeping this here for reference. Can delete if needed.

File: src/config/theme/index.ts (hunk 4 above)
Comment (AllTwos) [reply src/config/theme/index.ts:31]: Thanks!

**Pull request merged**